src/test/scala/edu/ie3/osmogrid/lv/region_coordinator/LvRegionCoordinatorIT.scala

```diff
@@ -0,0 +1,133 @@
+          LvRegionCoordinator()
+        )
+
+        testKit.run(
+          LvRegionCoordinator.Partition(
+            osmoGridModel = osmoGridModel,
+            administrativeLevel = adminLevel,
+            lvConfig = lvConfig,
+            replyTo = replyTo.ref
+          )
+        )
+
+        val messages = Range(0, 3).map { _ =>
+          val spawned = testKit
+            .expectEffectType[SpawnedAnonymous[LvRegionCoordinator.Request]]
+          val spawnedInbox = testKit.childInbox(spawned.ref)
+
+          spawnedInbox.receiveMessage() match {
+            case partitionMsg: LvRegionCoordinator.Partition =>
+              partitionMsg
+            case other => fail(s"Received unexpected message $other")
+          }
+        }
+        val models = messages.map(_.osmoGridModel)
+
+        messages.foreach(
+          _.administrativeLevel shouldBe BoundaryAdminLevel.AmtLevel
+        )
+        messages.foreach(_.lvConfig shouldBe lvConfig)
+        messages.foreach(_.replyTo shouldBe replyTo.ref)
+
+        // Recklinghausen
+        models.exists { m =>
+          m.buildings.size == 88 &&
+          m.highways.size == 19 &&
+          m.landuses.size == 17 &&
+          m.boundaries.map(_.entity.id).toSet.seq.equals(Set(62770, 56664)) &&
+          m.existingSubstations.size == 0
+        } shouldBe true
+
+        // Dortmund
+        models.exists { m =>
+          m.buildings.size == 318 &&
+          m.highways.size == 128 &&
+          m.landuses.size == 27 &&
+          m.boundaries
+            .map(_.entity.id)
+            .toSet
+            .seq
+            .equals(Set(10035847, 1829065)) &&
+          m.existingSubstations.size == 2
+        } shouldBe true
+
+        // Bochum
+        models.exists { m =>
+          m.buildings.size == 24 &&
+          m.highways.size == 24 &&
+          m.landuses.size == 22 &&
+          m.boundaries.map(_.entity.id).toSet.seq.equals(Set(1647366, 62644)) &&
+          m.existingSubstations.size == 0
+        } shouldBe true
+      }
+    }
+
+    "having reached the last administrative level" should {
+      "start MunicipalityCoordinators with the results" in {
+        val adminLevel = BoundaryAdminLevel.CountyLevel
+        val replyTo = TestProbe[LvRegionCoordinator.Response]()
+
+        val testKit = BehaviorTestKit(
+          LvRegionCoordinator()
+        )
+
+        val lvConfigCapped = lvConfig.copy(
+          boundaryAdminLevel =
+            lvConfig.boundaryAdminLevel.copy(lowest = adminLevel.osmLevel)
+        )
+
+        testKit.run(
+          LvRegionCoordinator.Partition(
+            osmoGridModel = osmoGridModel,
+            administrativeLevel = adminLevel,
+            lvConfig = lvConfigCapped,
+            replyTo = replyTo.ref
+          )
+        )
+
+        Range(0, 3).foreach { _ =>
+          testKit
+            .expectEffectType[SpawnedAnonymous[MunicipalityCoordinator.Request]]
+        }
+      }
+    }
+  }
+
+}
```

Comment:
Thanks for doing this very elaborate IT test and providing the elaborate it test and io framework - this is super handy 🙂